code/modules/economy/shady_robot.dm

```diff
@@ -83,7 +83,8 @@
 	/datum/commodity/contraband/briefcase,
 	/datum/commodity/bodyparts/heart,
 	/datum/commodity/bodyparts/r_eye,
-	/datum/commodity/bodyparts/l_eye)
+	/datum/commodity/bodyparts/l_eye,
+	/datum/commodity/sketchy_press_upgrade)
```

Comment:
I assume this was supposed to be added as a part of the printer press patch?

Reply:
yeah to that one